Pipfile

```diff
@@ -7,11 +7,12 @@ name = "pypi"
 requests = ">=2.20.0"
 gunicorn = "*"
 Flask = "*"
-rad = {git = "https://github.com/ManageIQ/aiops-rad.git",ref = "v0.8.2"}
+rad = {editable = true,git = "https://github.com/parsahosseini/aiops-rad.git",ref = "isolationforest"}
```

Comment:
will update this to use `v0.9.1` once the dependent `rad` PR is merged and released